GViewCore/src/App/Instance.cpp

```diff
@@ -11,8 +11,7 @@ constexpr uint32 MIN_CACHE_SIZE        = 0x10000;  // 64 K
 constexpr uint32 GENERIC_PLUGINS_CMDID = 40000000;
 constexpr uint32 GENERIC_PLUGINS_FRAME = 100;
 
-struct _MenuCommand_
-{
+struct _MenuCommand_ {
```

Comment:
undefined

Reply:
fixed in 95d3c2c669b3faf3d44b106626e4ec8aea86d74b